tools/OpenConsole.psm1

```diff
@@ -363,16 +363,91 @@ function Invoke-ClangFormat {
     }
 }
 
+function Invoke-StripBOM {
+    [CmdletBinding()]
+    Param (
+        [Parameter(Mandatory=$true,ValueFromPipeline=$true)]
+        [string]$Path
+    )
+
+    $files = Get-ChildItem -Path "$Path\*" -Include *.xaml
+    foreach ($file in $files) {
+        $content = Get-Content $file
+        [IO.File]::WriteAllLines($file.FullName, $content)

```

Comment:
I believe that this will explode at full speed if somebody is using Windows PowerShell to run our module (which many people I suspect do!). The default output format for Windows PS is _UTF-16_

Reply:
We need to explicitly set the output encoding when we do this

Reply:
wait but I only use Windows Powershell and that didn't explode for me. Unless it's because the Terminal is setting utf-8 by default, but I thought we weren't doing that...

Reply:
Ah, nevermind. WriteAllLines is unique of course in that it SPECIFICALLY writes u8-noBOM

Reply:
_of course_